src/utils/update/beneficial_owners_managing_officers_data_fetch.ts

```diff
@@ -34,13 +35,17 @@ const initialiseBoAndMoUpdateAppData = (appData: ApplicationData) => {
 };
 
 export const retrieveBeneficialOwners = async (req: Request, appData: ApplicationData) => {
+  const transactionId: string = appData.transaction_id as string;
+  const overseasEntityId: string = appData.overseas_entity_id as string;
   const pscs: CompanyPersonsWithSignificantControl = await getCompanyPsc(req, appData[EntityNumberKey] as string);
+  const boPrivateData: BeneficialOwnersPrivateData = await getBeneficialOwnerPrivateData(req, transactionId, overseasEntityId);
   if (pscs) {
     for (const psc of (pscs.items || [])) {
       logger.info("Loaded psc " + psc.kind);
       if (psc.ceasedOn === undefined) {
         if (psc.kind === "individual-beneficial-owner") {
           const individualBeneficialOwner = mapPscToBeneficialOwnerTypeIndividual(psc);
+          individualBeneficialOwner.usual_residential_address = mapPrivateAddress(boPrivateData, individualBeneficialOwner.ch_reference as string);
```

Comment:
ch_reference should not be undefined but I'm not sure we should just use "as string".

Reply:
ch_reference in the interface established prior has '?', should we take this out or should I keep it as it was ?

Reply:
Don't change interface please